src/Automation/Data/Config.cs

```diff
@@ -39,6 +39,11 @@ public class Config
         ///  <summary>The path to a file containing configuration instructing Axe Windows how to interpret custom UI Automation data.</summary>
         public string CustomUIAConfigPath { get; private set; }
 
+        /// <summary>
+        /// Determines whether multiple UIA roots will be scanned or just the first one.
+        /// </summary>
+        public bool EnableMultipleScanRoots { get; private set; }
```

Comment:
The name here suggests an action, not a state. Maybe `AreMultipleScanRootsEnabled` or something similar?